flink-table/flink-table-common/src/main/java/org/apache/flink/table/expressions/ValueLiteralExpression.java

```diff
@@ -106,7 +106,7 @@ else if (valueClass == Long.class && clazz == Duration.class) {
 
 		else if (valueClass == Period.class && clazz == Integer.class) {
 			final Period period = (Period) value;
-			convertedValue = period.toTotalMonths();
+			convertedValue = (int)period.toTotalMonths();
```

Comment:
```suggestion
			convertedValue = (int) period.toTotalMonths();
```
Please add a space after the casting.

Reply:
Thank you very much @wuchong. I have fix it.